include/spirv/unified1/spirv.core.grammar.json

```
@@ -11986,6 +11986,66 @@
           "value" : 6140,
           "capabilities" : [ "VectorComputeINTEL" ],
           "version" : "None"
+        },
+        {
+          "enumerant" : "ConduitKernelArgumentINTEL",
+          "value" : 6175,
+          "capabilities" : [ "FPGAArgumentInterfacesINTEL" ],
+          "version" : "None"
+        },
+        {
+          "enumerant" : "RegisterMapKernelArgumentINTEL",
+          "value" : 6176,
+          "capabilities" : [ "FPGAArgumentInterfacesINTEL" ],
+          "version" : "None"
+        },
+        {
+          "enumerant" : "MMHostInterfaceAddressWidthINTEL",
+          "value" : 6177,
+          "capabilities" : [ "FPGAArgumentInterfacesINTEL" ],
+          "version" : "None"
+        },
```

It has a parameter as well as the others below.
```suggestion
          "enumerant" : "MMHostInterfaceAddressWidthINTEL",
          "value" : 6177,
          "parameters" : [
            { "kind" : "LiteralInteger", "name" : "'AddressWidth'" }
          ],
          "capabilities" : [ "FPGAArgumentInterfacesINTEL" ],
          "version" : "None"
        },
```

---

done